utils/weather.js

```diff
@@ -12,5 +12,11 @@ module.exports = async location => {
         (select woeid from geo.places(1) where text="${location}")`
     }
   });
-  return results.data.query.results.channel.item;
+  const {
```

Comment:
nice, if we're going to use stuff like destructuring we might not have support in earlier versions of node. Do we require a minimum version?

If support is not available for earlier versions _and_ we dont want to require a min version we should consider implementing something like babel in a future ticket.

Reply:
I'd appreciate using modern javascript so babel would be good.

Reply:
I'm okay with either approach. I'll add a card to Trello for someone to take this on.